Code/GraphMol/FileParsers/MolSGroupWriting.cpp

```diff
@@ -511,15 +511,22 @@ std::string FormatV3000StringPropertyBlock(const std::string &prop,
   if (sgroup.getPropIfPresent(prop, propValue)) {
     if (!propValue.empty()) {
       ret << ' ' << prop << '=';
-      bool hasSpaces = propValue.find(' ') != std::string::npos;
+      bool needsQuotes = propValue.find(' ') != std::string::npos ||
+                         propValue.find('"') != std::string::npos;
```

Comment:
This check for double quotes is broader than what the specification says, but I think it should not be a problem (enclosing double quotes will be stripped away at parsing):
![image](https://user-images.githubusercontent.com/7498185/106285226-36028400-6212-11eb-82a1-918d73922e5e.png)

I think it is also a small change, but would you mind adding the case of the string starting with an opening parenthesis?

Reply:
stupid me for applying logic to the problem ;-)
I will implement what is in the spec.

Reply:
ok, I did a bit more investigation and it turns out that neither our parser nor BIOVIA Draw will correctly parse `FIELDINFO=foo""`, but both are fine with `FIELDINFO="foo"""`. 
BIOVIA Draw also parses `FIELDINFO=foo(bar)` as just `foo`.
BIOVIA Draw and Marvin Sketch are both fine with fields that don't need to be quoted starting with double quotes

So I think we should always include the (theoretically) extra starting double quotes for cases where the property includes either a double quote or a left paren.

I will update the PR to recognize the case with the paren as well.